swagger.yaml

```diff
@@ -4011,6 +4018,7 @@ definitions:
       - installation-disk-speed-check
       - container-image-availability
       - domain-resolution
+      - empty
```

Comment:
Why is it needed ?

Reply:
since we define an empty command, and it is sent to the agent, it is helpful that there will be a type assigned to it. For example it helps during verifications in unit test